backends/advanced/src/advanced_omi_backend/models/job.py

```
@@ -63,6 +66,11 @@ async def _ensure_beanie_initialized():
             _beanie_initialized = True
             logger.info("✅ Beanie initialized in RQ worker process")
 
+            # Register prompt defaults (needed for title/summary generation etc.)
+            prompt_registry = get_prompt_registry()
+            register_all_defaults(prompt_registry)
+            logger.info("✅ Prompt registry initialized in RQ worker process")
+
         except Exception as e:
             logger.error(f"❌ Failed to initialize Beanie in RQ worker: {e}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)